app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -228,7 +390,7 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
     }
 
     override fun setupDeviceAdapter(scanList: List<String>) {
-        Timber.d("Connected Successfully")
+        Timber.e("Connected Successfully")
```

Comment:
same, it should be a debugging message, not an error.

Reply:
Updated